sdk/core/Azure.Core/src/Shared/BearerTokenChallengeAuthenticationPolicy.cs

```diff
@@ -46,9 +47,9 @@ internal BearerTokenChallengeAuthenticationPolicy(TokenCredential credential, IE
         }
 
         /// <inheritdoc />
-        public override ValueTask ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPipelinePolicy> pipeline)
+        public override async ValueTask ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPipelinePolicy> pipeline)
         {
-            return ProcessAsync(message, pipeline, true);
+            await ProcessAsync(message, pipeline, true).ConfigureAwait(false);
```

Comment:
please don't add await here. This policy is on the hot path and we should reduce the state machine count.

Reply:
Thanks, I meant to change this back after the most recent changes.